kustomization/components/kube-vip/rbac.yml

```diff
@@ -23,9 +23,9 @@ rules:
       - endpoints
       - services
     verbs:
+      - endoints
```

Comment:
Typo?

Reply:
Not from me, it's not!

https://github.com/kube-vip/kube-vip/blob/7d7036fae92ffe32f0a04de7f834a0fb7e457ce2/pkg/kubevip/config_generator.go#L103

Reply:
We'll want to wait for https://github.com/kube-vip/kube-vip/pull/1141 getting released before we want to merge this, alas.